<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNoiseProcDlg</class>
 <widget class="QDialog" name="CNoiseProcDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noise Processing Setup</string>
  </property>
  <widget class="QGroupBox" name="groupBoxNBlanker">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Noise Blanker</string>
   </property>
   <widget class="QCheckBox" name="checkBoxNBOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
   <widget class="CSliderCtrl" name="frameNBThresh">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="CSliderCtrl" name="frameNBWidth">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSliderCtrl</class>
   <extends>QFrame</extends>
   <header>gui/sliderctrl.h</header>
   <container>1</container>
   <slots>
    <slot>valueChanged(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxNBOn</sender>
   <signal>toggled(bool)</signal>
   <receiver>CNoiseProcDlg</receiver>
   <slot>OnNBOn(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnNBOn(bool)</slot>
 </slots>
</ui>
